std_msgs/Header header
int16[] instances
geometry_msgs/Point[] centroids
std_msgs/Float32MultiArray features